lib/structures/Member.js

```diff
@@ -76,6 +82,11 @@ class Member extends Base {
         }
     }
 
+    get voiceState() {
+        const voiceState = this.guild.voiceStates.get(this.id);
+        return voiceState ? voiceState : new VoiceState({id: this.id});
```

Comment:
How about merging these into 1 line? Would probably look a lot cleaner in my opinion.
```js
return this.guild.voiceStates.get(this.id) || new VoiceState({ id: this.id })
```

Reply:
https://github.com/abalabahaha/eris/pull/548#discussion_r340955586